django_netjsonconfig/widgets.py

```diff
@@ -27,8 +27,8 @@ def media(self):
     def render(self, name, value, attrs={}):
         attrs['class'] = 'vLargeTextField jsoneditor-raw'
         html = """
-<a href="#advanced_editor" class="button json-editor-btn-edit normal-mode">{0}</a>
```

Comment:
Yesterday I left this reply on the mailing list:
https://groups.google.com/d/msg/openwisp/aP5pJgBH4ZQ/pP47lTQpCAAJ

> Could you hook the event to the new "exit advanced mode" button?
> 
> That way we would avoid having redundant code sitting around and risking of being forgotten. Removing it will make the job of future maintainers easier.
> 
> Have you ever heard of "the broken windows theory"?
> 
> If not, look it up, it's an interesting concept.
> 
> Consider redundant code like a broken window.
> 
> Federico